src/bp-templates/bp-nouveau/js/buddypress-messages.js

```diff
@@ -1759,6 +1759,11 @@ window.bp = window.bp || {};
 					errors.push( 'message_content' );
 				}
 
+				this.model.set('message_content', this.model.get('message_content').replace(/&nbsp;/g, '').trim(), { silent: true });
+				if ($(this.model.get('message_content')).text().trim() === '') {
+					errors.push('message_content');
+				}
+
```

Comment:
@buddy4kp Here too, we have one validation code few lines below this to check for other toolbar options like media, document etc. if they are added then content should be allowed empty.